### PR TITLE
added baselineOffset concept to SwiftUI viewmodifier terms

### PR DESCRIPTION
**Pull request by jfelix9**
### Description

Closes [#1569](https://github.com/Codecademy/docs/issues/1569) . Added a the .baselineOffset() view modifier term under SwiftUI with an image.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/swiftui/concepts/viewmodifier/terms/baselineOffset/baselineOffset.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+```pseudo
+Text("This text is shifted up.")
+    .baselineOffset(10)
+Text("This text is shifted down.")
+    .baselineOffset(-10)
+```
+
+The `.baselineOffset()` modifier method uses the following parameter:
+
+- `baselineOffset` specifies how many points the text is shifted up or down. The following options are possible:
```
Comment (SSwiniarski) [inline content/swiftui/concepts/viewmodifier/terms/baselineOffset/baselineOffset.md:29]: This line and the following table is misleading, implying that 0,5,-5 are the only valid options. Simply say the parameter can be positive, negative or zero.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@jfelix9 I did a first review and had a couple of comments.

**Comment by jfelix9** [comment]
Hi @SSwiniarski I fixed the grammatical mistakes and replaced the misleading parts.

File: content/swiftui/concepts/viewmodifier/terms/baselineOffset/baselineOffset.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.baselineOffset()`** modifier method moves text in a view [view](https://www.codecademy.com/resources/docs/swiftui/views) vertically relative to its baseline. The bounds of the text's parent view expands to contain the modified text.
```
Comment (SSwiniarski) [inline content/swiftui/concepts/viewmodifier/terms/baselineOffset/baselineOffset.md:16]:
```suggestion
The **`.baselineOffset()`** modifier method moves text in a view [view](https://www.codecademy.com/resources/docs/swiftui/views) vertically relative to its baseline. The bounds of the text's parent view expands to contain the modified text.
```
```suggestion
The **`.baselineOffset()`** modifier method moves text in a [view](https://www.codecademy.com/resources/docs/swiftui/views) vertically relative to its baseline. The bounds of the text's parent view expands to contain the modified text.
```

File: content/swiftui/concepts/viewmodifier/terms/baselineOffset/baselineOffset.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+Text("This text is shifted up.")
+    .baselineOffset(10)
+Text("This text is shifted down.")
+    .baselineOffset(-10)
+```
+
+The `.baselineOffset()` modifier method uses the following parameter:
+
+- `baselineOffset` specifies how many points the text is shifted up or down. The parameter can be positive, negative, or zero.
+
```
Comment (SSwiniarski) [inline content/swiftui/concepts/viewmodifier/terms/baselineOffset/baselineOffset.md:30]:
```suggestion
```
remove line break between bullet points.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@jfelix9 I had a couple more comments.

**Comment by jfelix9** [comment]
Hi @SSwiniarski , I've implemented the recommended changes, ready for review

**Review by SSwiniarski — APPROVED** [review]
@jfelix9 looks good for a second review.

File: content/swiftui/concepts/viewmodifier/terms/baselineOffset/baselineOffset.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            .border(Color.yellow)
+            
+        Text("I have 15 points of vertical shift upwards.")
+            .baselineOffset(15)
+            .border(Color.blue)
+            
+        Text("I have 15 points of vertical shift downwards.")
+            .baselineOffset(-15)
+            .border(Color.green)
+    }
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/terms/baselineOffset/baselineOffset.md:47]: I nested this in a stack. Can you include the background color you used?

**Comment by jfelix9** [comment]
Hi @yangc95 I've added the color and padding to the stack to have it match the image

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
🙌 Neat! @jfelix9 live link: https://www.codecademy.com/resources/docs/swiftui/viewmodifier/baselineOffset

Screenshot 2023-05-01 at 4 32 44 PM

cc: @SSwiniarski @KTom101

**Pull request merged**